tests/SqlFormatterTest.php

```diff
@@ -22,34 +22,38 @@ final class SqlFormatterTest extends TestCase
     /** @var string[] */
     private $sqlData;
 
+    /** @var SqlFormatter */
+    private static $formatter;
+
     public static function setUpBeforeClass() : void
     {
+        self::$formatter = new SqlFormatter();
```

Comment:
What about using `setUp()` instead of `setUpBeforeClass` ?

Reply:
I did that initially, but then there is a test about cache stats that failed. We have to use the same instance for every test for it to stay green. Or rework that test, which can be done later (the goal of this PR is to remove statism from src, as a BC-break).